tests/features/site-branding.feature

```diff
@@ -72,3 +72,41 @@ Feature: Site branding
     Then I should see the "page header" element in the "page"
     When I am on "the user registration page"
     Then I should see the "page header" element in the "page"
+
+  @javascript @enable-non-eu-language
```

Comment:
This is a refactoring of the test from EWPP-1854. Since we need to ensure correctness of the data in the logo in different languages we need to pass them into the Behat context.